cmd/manager/main.go

```diff
@@ -58,7 +59,10 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	log.Info("setting up manager")
-	mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0"})
+	mgr, err := manager.New(cfg, manager.Options{
+		Metrics: server.Options{
+			BindAddress: "0.0.0.0",
```

Comment:
This looks sane yes? idk why it was `0` before

Reply:
```
	// BindAddress is the bind address for the metrics server.
	// It will be defaulted to ":8080" if unspecified.
	// Set this to "0" to disable the metrics server.
```

Reply:
Ah thank you!